torch/_dynamo/variables/user_defined.py

```diff
@@ -570,7 +570,7 @@ class KeyedJaggedTensorVariable(UserDefinedObjectVariable):
     def is_matching_object(obj):
         try:
             from torchrec.sparse.jagged_tensor import KeyedJaggedTensor
-        except ImportError:
+        except (ImportError, AttributeError):
```

Comment:
How

Reply:
This happens when you have torchrec but some old version

Reply:
The easiest way to reproduce this is accidentally install fbgemm-gpu-nightly at the same time as a source FBGEMM install, or perhaps with an incompatible source build of pytorch